rsmtool/preprocessor.py

```diff
@@ -404,7 +397,8 @@ def trim(values,
             trimming the raw regression predictions.
         tolerance : float, optional
             The tolerance that will be used to compute the
-            trim interval. Defaults to 0.4998.
+            trim interval.
+            Defaults to 0.4998.
```

Comment:
Stylistically, I think numbers should be treated as literals, but I won't comment on any more instances. Just a thought.

```suggestion
            Defaults to ``0.4998``.
```